modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,57 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Optional: Enabling monitoring of event listeners for Triggers for user-defined projects
+
+On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`. Cluster administrators can configure the monitoring of event listener metrics for Triggers in a user-defined project. When configured, users and administrators can view the metrics in the OpenShift web console.
+
+[discrete]
+.Prerequisites
+
+* Install the {pipelines-title} Operator.
+* Enable monitoring for user-defined projects.
+
+[discrete]
+.Procedure
+
+. For each event listener, create a service monitor. For example, to view the metrics for the `github-listener` event listener in the `test` namespace, create the following service monitor:
++
+[source,yaml]
```

Comment:
Should they create exactly the service monitor shown, or is this just one possible example? If it is an example, how about saying:
```suggestion
. For each event listener, create a service monitor. For example, to view the metrics for the `github-listener` event listener in the `test` namespace, create a service monitor. For example:
+
[source,yaml]
```
OR
```suggestion
. For each event listener, create a service monitor. For example, to view the metrics for the `github-listener` event listener in the `test` namespace, create a service monitor.
+
.Example service monitor
[source,yaml]
```